CATALOG.md

```diff
@@ -35,3 +35,71 @@ nativefier https://www.udemy.com/
 ### Notes
 
 Most videos will work, but to be sure everything works you are better off using the `--widevine` version AND signing the app afterwards. See this post: [#1147 (comment)](https://github.com/nativefier/nativefier/issues/1147#issuecomment-828750362)
+
+
+## Spotify
+
+```
+nativefier https://open.spotify.com/
+--name "Spotify" 
+--widevine 
+-u "<useragent your browser uses>"
+--inject spotify.js
+--inject spotify.css
+--icon icon.
+```
+
+### Notes
+
+- [Inject](https://github.com/nativefier/nativefier/blob/master/docs/api.md#inject) the following javascript as `spotify.js` to prevent "Unsupported Browser" messages.
+```javascript
+function dontShowBrowserNoticePage() {
+    const browserNotice = document.getElementById('browser-support-notice');
+    console.log({ browserNotice })
+    if (typeof browserNotice !== "undefined" && browserNotice !== null) {
+        document.getElementsByTagName('body')[0].style.display = 'none';
+    }
+}
+
+function reload() {
+    window.location.href = window.location.href;
+}
+
+function nukeWorkers() {
+    dontShowBrowserNoticePage();
+    if ('serviceWorker' in navigator) {
```

Comment:
Useless as always true in Nativefier, let's drop the `if`

Reply:
> Useless as always true in Nativefier, let's drop the `if`
Are you sure? I think this is only true when a serviceWorker has been spun up (though I could be wrong).

Reply:
@TheCleric nope it's always present, if supported by the browser: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/serviceWorker